prboom2/src/SDL/i_system.c

```diff
@@ -333,6 +333,15 @@ dboolean HasTrailingSlash(const char* dn)
           );
 }
 
+static const char *GetExecutableDir(void)
```

Comment:
We shouldn't have 2 fiunctions with essentially the same name that do different things :^)
Let's call this `I_GetBasePath` to correspond to the SDL semantics. Probably `I_DoomExeDir` needs a better name but that's a separate topic.